eng/source-build-patches/0008-Remove-import-of-PackageVersionProps-from-eng-Vervsi.patch

```diff
@@ -0,0 +1,24 @@
+From 42bb5cf46bbc21011822afeabda30eaee72d25cd Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Mon, 20 Apr 2020 15:09:58 +0000
+Subject: [PATCH] Remove import of PackageVersionProps from eng/Vervsions.props
+ It should be imported by Arcade SDK
+
+---
+ eng/Versions.props | 2 --
+ 1 file changed, 2 deletions(-)
+
+diff --git a/eng/Versions.props b/eng/Versions.props
+index cc4a7b62914..cef8de452b4 100644
+--- a/eng/Versions.props
++++ b/eng/Versions.props
+@@ -167,6 +167,4 @@
+     <MicrosoftNETRuntimeEmscripten2021Nodewinx64Version>6.0.0-preview.6.21275.1</MicrosoftNETRuntimeEmscripten2021Nodewinx64Version>
+     <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETRuntimeEmscripten2021Nodewinx64Version)</MicrosoftNETRuntimeEmscriptenVersion>
+   </PropertyGroup>
+-  <!-- Override isolated build dependency versions with versions from Repo API. -->
+-  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
```

Comment:
@ViktorHofer this is already done by Arcade for product build. The only case where I don't see this being true is when this gets imported from tests. Do you know why we need to import it or if we still do? It comes from https://github.com/dotnet/corefx/pull/37681/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67

Reply:
That can just be removed, it's an artifact of how source build worked before.

Reply:
https://github.com/dotnet/runtime/pull/53986